src/routes/relay/services/relay-limit.service.ts

```diff
@@ -36,6 +41,11 @@ export class RelayLimitService {
     remaining: number;
     expiresAt?: number;
   } {
+    this.loggingService.log(
+      'Called getRelayLimit for address %s on chain %s',
```

Comment:
nitpick: Maybe "Getting the relay limit for address %s on chain %s" instead of "Called ..." just so that the logs are less tied to the "function calls" but more towards the actual functionality/flow.